cmd/controlloop/controlloop.go

```diff
@@ -56,6 +61,26 @@ func main() {
 	}
 
 	networkController.Start(stopChan)
+
+	s := gocron.NewScheduler(time.UTC)
```

Comment:
do we want the timezone to be configurable ? I'm OK with forcing everyone to use UTC.

Reply:
We could change this down the line if a user finds it important. I'm indifferent but I figured UTC is standard.